workshop/content/lab1.4_developer.md

```diff
@@ -39,7 +41,7 @@ Service 'hello-python' created to latest revision 'hello-python-dydsc-1' is avai
 http://hello-python-hello.apps.cluster-tysons-4d23.tysons-4d23.example.opentlc.com
 ```
 
-5.  Test it
+4.  Test it
 
 In one terminal, watch the pods:
 ```
```

Comment:
In another terminal, curl the endpoint:

```
HELLO_URL=$(oc get route.serving.knative.dev hello-python --template='{{.status.url}}')
curl $HELLO_URL
```

Reply:
In another terminal we'll curl the endpoint again.

```
curl $HELLO_URL
```

Reply:
👍